docs/message.md

```diff
@@ -12,8 +15,15 @@ these details can be separated from the actual requests.
 $ composer require php-http/message
 ```
 
+## Authentication
```

Comment:
I agree: roughly equivalent to the [php-http/message directory listing](https://github.com/php-http/message/tree/master/src).